src/R/Components/Impl/ConnectionManager/Implementation/ConnectionManager.cs

```diff
@@ -247,27 +263,11 @@ internal class ConnectionManager : IConnectionManager {
                 }
             }
 
-            if (!string.IsNullOrEmpty(connectionInfo?.Path) && await TrySwitchBrokerAsync(connectionInfo)) {
-                return;
-            }
-
-            var connection = RecentConnections.FirstOrDefault(c => !c.IsRemote);
```

Comment:
Ok

---
In reply to: [80280455](https://github.com/Microsoft/RTVS/pull/2393#discussion_r80280455) [](ancestors = 80280455)